policy.go

```diff
@@ -1258,33 +840,51 @@ func executePolicySession(tpm *tpm2.TPMContext, policySession tpm2.SessionContex
 //
 // On success, subsequent calls to SealedKeyObject.UnsealFromTPM will fail with a ErrSealedKeyAccessLocked error until the next TPM
 // restart or TPM reset.
-func LockAccessToSealedKeys(tpm *TPMConnection) error {
+//
+// TODO: Update this documentation and maybe rename the function, as it doesn't make much sense now.
+func LockAccessToSealedKeys(tpm *TPMConnection, pcrs []int) error {
```

Comment:
maybe something along "BlockPCRProtectionsPolicies" or similar ?

Reply:
Yeah, I think this is ok.